tests/nodeos_forked_chain_test.py

```diff
@@ -164,10 +164,11 @@ def getMinHeadAndLib(prodNodes):
 
     # ***   setup topogrophy   ***
 
-    # "bridge" shape connects defprocera through defproducerk (in node0) to each other and defproducerl through defproduceru (in node01)
+    # "bridge" bridge_consecutive_shape.json shape connects defproducera through defproducerk (in node0) to each other
+    # and defproducerl through defproduceru (in node01)
     # and the only connection between those 2 groups is through the bridge node
 
-    if cluster.launch(prodCount=prodCount, topo="bridge", pnodes=totalProducerNodes,
+    if cluster.launch(prodCount=prodCount, topo="./tests/bridge_consecutive_shape.json", pnodes=totalProducerNodes,
```

Comment:
Created a bridge shape file for this test that matched the consecutive layout of block producers as expected by the test. The description of the test clearly indicates it wants to create a set of producers split from one another completely so that once one set starts it can producer a long chain by itself.